docs/gwcs/developer_introduction.rst

```diff
@@ -0,0 +1,195 @@
+.. _gwcs:
```

Comment:
```suggestion
.. _dev_intro:
```
keep `index.rst` as the `_gwcs`. That way intersphinx links will continue pointing at the same page.